lib/main.dart

```diff
@@ -82,8 +83,11 @@ class _MyHomePageState extends State<MyHomePage> {
                         _game = _createGame(_grid.value);
                       });
                     },
+                    style: ElevatedButton.styleFrom(
+                        backgroundColor: Colors.amber[600]),
```

Comment:
Create file colors.dart and define color constant here, so that we can observe and fine tune the colors in one place.
Move other hard coded colors here too.

Reply:
done, also split colors by widgets